Azure-Samples/azure-search-java-samples
# Quickstart sample Java 11 update

## Purpose

Consistent with other samples. 

Most changes relate to updating to `HttpClient` for request / response.

Broader use of `var` and `final var`. 

Reformatted consistently and optimized imports.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[X ] Code style update (formatting, local variables)
[X ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Modify src/main/resources/azure_search_config with service name and API Key
* Execute Maven goal `verify exec:java`


* Test the code

`exec:java` goal will execute `com.microsoft.azure.search.samples.demo.App.main` 

You should see build and then program output: build and query index. 

## What to Check
Verify that the following are valid :

* Can be run from command-line Maven? `mvn verify exec:java` (If Maven installed)
* Can be imported and run from VSCode? (Requires Maven extension)
* Can be imported and run from IDEA? 

@LisaLeib @HeidiSteen
